docker-compose/integrator-broker-analytics/README.md

```diff
@@ -27,7 +27,7 @@
      Instead, extract the zip file and directly browse to `docker-ei-<released-version>/docker-compose/integrator-broker-analytics` folder. 
      
      > If you are to try out an already released tag, after executing 2nd step, checkout the relevant tag, 
-     i.e. for example: <br> git checkout tags/v6.1.1.4 and continue below steps.
+     i.e. for example: <br> git checkout tags/v6.2.0.4 and continue below steps.
```

Comment:
Shouldn't this be v6.2.0.1 ? Please fix all places.

Reply:
+1

Reply:
For WSO2 Enterprise Integrator For Integration, Broker and Business Process (BPS) Use-cases With Analytics Support deployment, this fix would be available in https://github.com/wso2/docker-ei/pull/51.